resources/css/style.css

```diff
@@ -0,0 +1,103 @@
+/* stylelint-disable function-url-quotes */
+/* stylelint-disable font-family-name-quotes */
```

Comment:
It is recommended that we follow the good practices and if we code following a code of conduct. For this reason, it is not advisable to deactivate linters.

- [ ] Kindly, try to work around it without deactivating linters